test/DacpacTool.Tests/ExtensionsTest.cs

```diff
@@ -73,7 +73,121 @@ public void CanGetModelValidationErrors()
 
             var error = modelValidationErrors.First();
             error.Severity.ShouldBe(Microsoft.SqlServer.Dac.Model.ModelErrorSeverity.Error);
-            error.ToString().ShouldBe("../../../../TestProjectWithErrors/Procedures/csp_Test.sql(2,18):ModelValidationError Error SQL71501: SqlSubroutineParameter: [dbo].[csp_Test].[@p_Parameter] has an unresolved reference to SqlBuiltInType [dbo].[MyCustomType].");
+            error.ErrorCode.ShouldBe(71501);
```

Comment:
Did we remove the assertion on ToString() for a reason? I think that still has value to avoid regressions in the output we give.

Reply:
If we has localized `.NET` on machine running tests it report error message other than that with which you are compare. So I think that we must check `ModelValidationError` properties to check correct result, instead of check `ToString`